setup.cfg

```diff
@@ -23,18 +23,18 @@ zip_safe = False
 package_dir=
     =src
 packages=find_namespace:
-python_requires = >=3.9
+python_requires = >=3.11
 include_package_data = True
 
 install_requires =
-    uvicorn
-    fastapi
-    fastapi-versioning
-    asgi-correlation-id
-    cryptography
-    boto3
-    geojson>=3.0.0
-    python-json-logger>=2.0.0
+    uvicorn==0.31
+    fastapi==0.115
+    fastapi-versioning==0.10
+    asgi-correlation-id==4.3
+    cryptography==43.0
+    boto3==1.35
+    geojson==3.1
+    python-json-logger==2.0
```

Comment:
Does it have to pin to a specific version? Can we do `>=`?

Reply:
The intent here is that we only get bug/security/maintenance fixes automatically and major/minor upgrades have to be deliberate.  Python does not have an equivalent to the npm `~` that I am aware of but I can add `.*` for more clarity.